schemas/schemas/adminQuery.json

```
@@ -8,6 +8,10 @@
         "description"
     ],
     "properties": {
+        "comment": {
+            "description": "Add the comment",
+            "type": "string"
```

I think we should allow comments to also be arrays or objects:
```suggestion
            "type": ["string", "array", "object"]
```

---

Did you want this for all comments?

---

What do you think? We can leave comments as just simple strings, which might be convenient in the future if we want to edit them programmatically, or we can allow comments to be arbitrary data (strings, objects, or arrays). The benefit of allowing them to be arbitrary data would be to allow flexibility for people to store other metadata into their JSON files (without having to encode it as a string). I don't really have a good use case for this at the moment, so it's more of a "why not allow it" sort of thing. What do you think? @nwalters512 @nicknytko any thoughts?

---

I guess I lean towards allowing arbitrary data here, but I don't feel strongly and I'm happy to be swayed to just "string".

---

I personally don't see why we shouldn't allow any data type here &mdash; one use case could be to have an array of string comments maybe?  Or for formatting purposes:

```
"comment": [
    "bla bla here are some bullet points",
    [
        "1. asdf",
        "2. qwerty",
        "3. yuiop"
    ]
]
```

---

In any case I agree that this is sort of a "why not" change, and it seems like it would be trivial enough to add as well.

---

Yeah, agreed. We don't need to emphasize this in the docs or examples. Just a brief note in the docs would be sufficient, I think.

---

Added brief mention in docs in d80069e1c

---

Looks good!